### PR TITLE
Lingjie's Variables exercises

### PR DESCRIPTION
**Pull request by clairewlj**

### CODE REVIEW

File: _posts/clairewlj/2016-01-18-Chapter2exercises.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+author: clairewlj
+layout: post
+title: Lingjie's variables exercises
```
Comment (eah13) [inline _posts/clairewlj/2016-01-18-Chapter2exercises.md]: I believe this apostrophe will bork your post.

**Comment by eah13** [comment]
Looks good once you get the post revised!

**Comment by clairewlj** [comment]
@eah13 So sorry about that. I add " quotes to the title. Thank you!

**Comment by eah13** [comment]
No problem! :rocket:

**Pull request merged**